### PR TITLE
Module 2 and 3 Typos

### PR DESCRIPTION
**Pull request by ArtichokeSap**
Changes to fix typos as detailed in [Issue #146](https://github.com/rsokl/Learning_Python/issues/146).

### CODE REVIEW

File: Python/Module3_IntroducingNumpy/VectorizedOperations.md [hunk 1]
```diff
@@ -686,10 +686,10 @@ Add the four quadrants of `x`, producing a shape-(2, 2) output.
...
 ...               [12, 13, 14, 15]])
 
-    # top-left  top-right    bottom-left  bottom-right
->>> x[:2, :2] + x[:2, -2:] + x[-2:, :2] + x[-2:, -2:]
-array([[20, 24],
-       [36, 40]])
+                    # top-left           top-right            bottom-left        bottom-right
+>>> np.array([[np.sum(x[:2, :2]), np.sum(x[:2, 2:])], [np.sum(x[2:, :2]), np.sum(x[2:, 2:])]])
+array([[10, 18],
+       [42, 50]])
```
Comment (rsokl) [inline Python/Module3_IntroducingNumpy/VectorizedOperations.md:692]: The original is correct. It is asking the reader to take each shape-(2, 2) quadrant and add them element-wise. Perhaps the wording should be updated to clarify this.


```suggestion
# top-left  top-right    bottom-left  bottom-right
>>> x[:2, :2] + x[:2, -2:] + x[-2:, :2] + x[-2:, -2:]
array([[20, 24],
       [36, 40]])
```

Comment (ArtichokeSap) [reply Python/Module3_IntroducingNumpy/VectorizedOperations.md:692]: I wasn't sure whether to change the question or the answer.  I couldn't think of a good way to make the question more clear.  Now that I'm re-reading it again, I see that it isn't _wrong_, but maybe still ambiguous.

Maybe "Produce the element-wise sum of each of the shape-(2,2) quadrants," though that seems pedantic.

Comment (rsokl) [reply Python/Module3_IntroducingNumpy/VectorizedOperations.md]: I don't think it is pedantic if it would prevent the reader from being confused.

**Comment by rsokl** [comment]
If you are all finished, I'll go ahead and merge

**Comment by ArtichokeSap** [comment]
I think that’s everything!

**Comment by rsokl** [comment]
🎉

**Pull request merged**